tmt-app/src/Components/MajorCareerList.jsx

```diff
@@ -25,7 +26,28 @@ function MajorCareerList() {
                 padding: "0.25rem 0",
               }}
             >
-              {item.name}
+              <Link
+                to={`/majors/${item.key}`}
+                component={RouterLink}
+                sx={{
+                  color: "Boba.main",
+                  textDecoration: "underline",
+                  "&:hover": {
+                    color: "RedBean.main",
+                    fontWeight: "bold",
+                  },
+                  "&::after": {
+                    display: "block",
+                    content: `"${item.name}"`,
+                    fontWeight: "bold",
+                    height: "0px",
+                    overflow: "hidden",
+                    visibility: "hidden",
```

Comment:
What is this style used for btw?

Reply:
Bold styling would change the sizing of the links. This one of the solutions to help preset the sizing so that the links would not grow and shrink the container when hovered